### DISCUSSION TITLE
Twist Optimization only

### DISCUSSION THREAD

**Discussion by shaheidula** [mdolab/dafoam#320] [General]
Dear Prof. Ping He, 
I tried to run the twist optimization only, however, the optimization process stays without any response at the end of the "running coloring solver" and showing "Warning: The link for attached point 0 was drawnfrom the curve to the end of the ray,indicating that the ray might not have been longenough to intersect the nearest curve.". 
Looking forward to hearing from you, 
[logOpt.txt](https://github.com/mdolab/dafoam/files/8948588/logOpt.txt)

**Comment by shaheidula** [comment]
With the previous version, I have run the twist optimization only without considering other design variables. But with this version, I am having this issue.

**Comment by friedenhe** [comment]
Check [here](https://dafoam.github.io/mydoc_get_started_faq.html#how-to-fix---warning-xx-points-not-projected-to-tolerance-1e-12-max-error--xxx--rms-error-xxx-list-of-points-is-pt-delta) and [also here](https://dafoam.github.io/mydoc_get_started_faq.html#how-to-fix-warning-xxx-the-ray-might-not-have-been-long-enough-to-intersect-the-nearest-curve)
